modules/functional-test/src/main/resources/config.properties

```diff
@@ -39,6 +39,7 @@ api-server-protocol=http
 api-version-string=v1
 user-name=admin
 password=admin
+test-user=wasabi_reader@example.com
```

Comment:
@jcwuzoegiver the above is a default user we need to have, because admin is the superuser and he should be able to assign users to the applications. for that he need one non-super admin user and hence i used wasabi_reader@example.com